easybuild/easyconfigs/b/BayPass/BayPass-2.1-foss-2015a.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'BayPass'
+version = '2.1'
+
+homepage = 'http://www1.montpellier.inra.fr/CBGP/software/baypass/'
+description = """The package BayPass is a population genomics software which is primarily aimed at identifying genetic markers subjected to selection and/or associated to population-specific covariates (e.g., environmental variables, quantitative or categorical phenotypic characteristics)."""
```

Comment:
@AFCJamie we try to limit line length to 120 characters or less, please break up this long line